tests/sklearn/test_sklearn_models.py

```diff
@@ -1139,6 +1140,56 @@ def check_load_fitted_sklearn_linear_models(model_class, n_bits, x, y, check_flo
     )
 
 
+def check_rounding_consistency(
```

Comment:
nice test! indeed having the MSB configurable here makes writing this test easy. 

But if the MSB is not configurable, then maybe you have a global variable say `TREE_COMPARISON_MSB` and you change that in the test before each compilation.

Reply:
I propose to add a function that readjust the auto_rounder according to a new msb value, can you check it ?